frontend/src/assets/stylesheets/components/footer.sass

```diff
@@ -0,0 +1,73 @@
+.footer
+  padding: 30px 0px 20px
+  margin-right: 40px
+
+  a
+    color: #A0A0A0
+    text-decoration: none
+    &:hover
+      color: #48bd66
+
+  .social-links
+    display: flex
+    justify-content: space-evenly
+    padding: 0
+    list-style-type: none
+
+    max-width: 1250px
+    margin: 16px auto
+
+    li
+      a
+        display: block
+        svg
+          margin: 14px
+          height: 22px
+
+          path
+            fill: #A0A0A0
+            transition: fill $transition-duration $transition-function
+        &:hover
+          &.-linkedin
```

Comment:
That's not very BEMish too :)

Reply:
Really, why?
I used modifiers to change behavior diffrents icons on hover

Reply:
Yep, you are right. My bad :)